db/rdb/rdb.go

```diff
@@ -62,21 +64,25 @@ func NewRDB(family, dbType, dbpath string, debugSQL bool) (driver *Driver, locke
 
 // NewOvalDB create a OvalDB client
 func (d *Driver) NewOvalDB(family string) error {
+	if _, ok := ovalMap[family]; ok {
+		return nil
+	}
+
 	switch family {
 	case c.Debian:
-		d.ovaldb = NewDebian()
+		ovalMap[c.Debian] = NewDebian()
```

Comment:
Why is new instance created every time? It seems to be better that this map should be created once.

Reply:
If ovalMap has instance, this function early return.

Reply:
I'm sorry that this line was not appropriate. As for RedHat, NewRedHat() is executed. After that, when CentOS is coming, NewRedHat will be executed again. Same as SUSE.
However, SUSE needs `family`, so this may be enough as it is now.